crates/common_utils/Cargo.toml

```diff
@@ -33,6 +33,8 @@ tokio = { version = "1.27.0", features = ["macros", "rt-multi-thread"], optional
 thiserror = "1.0.40"
 time = { version = "0.3.20", features = ["serde", "serde-well-known", "std"] }
 md5 = "0.7.0"
+phonenumber = "0.3.2+8.13.9"
+test-case = "3.1.0"
```

Comment:
Not sure if we wanna add this crate,
we could do some analysis to check how much value it adds compared to the additional compilation time/size etc

Anyways can you move this to dev-dependencies...